lightbeam/__main__.py

```diff
@@ -21,6 +21,7 @@ def emit(self, record):
 logger.setLevel(logging.getLevelName('INFO'))
 logger.addHandler(handler)
 # logging.basicConfig(handlers=[ExitOnExceptionHandler()])
+lock = False
```

Comment:
What does this line do? This lock won't be accessible within `Lightbeam`.

Reply:
Good point, this was a leftover from when I was (foolishly) considering using global variables to lock async processes when an auth token times out. Removed now.